Purchases/SubscriberAttributes/RCSubscriberAttribute.m

```diff
@@ -79,7 +80,7 @@ - (instancetype)initWithDictionary:(NSDictionary <NSString *, NSObject *> *)dict
 - (NSDictionary <NSString *, NSObject *> *)asBackendDictionary {
     return @{
         BACKEND_VALUE_KEY: self.value ?: @"",
-        BACKEND_TIMESTAMP_KEY: @(self.setTime.timeIntervalSince1970)
+        BACKEND_TIMESTAMP_KEY: @(self.setTime.millisecondsSince1970)
```

Comment:
the value of the key has also changed to `updated_at_ms`

Reply:
good catch, updated